.github/workflows/check_netty_snapshots.yml

```diff
@@ -3,7 +3,7 @@ name: Check Netty SNAPSHOTS
 on:
   schedule:
     - cron: "0 14 * * *"
-
+permissions: read-all
```

Comment:
this could also be enabled on `codeql` workflow. that said, ultimately the org-wide setting will probably be used on top of explicit read-only `permissions` clauses

Reply:
mmm for CodeQL we need `security-events: write`
see https://github.com/github/codeql-action#usage

Reply:
right. that doesn't seem too bad and probably better than the (current) default of write-all so I'd say go for it

Reply:
ok we have now

```
GITHUB_TOKEN Permissions
  Metadata: read
  SecurityEvents: write
```

https://github.com/reactor/reactor-netty/runs/4940728231?check_suite_focus=true#step:1:13